src/components/pages/Navigation.vue

```diff
@@ -105,22 +104,26 @@ header.theme--dark.v-sheet
 
 
 
-
-      <v-menu
-      offset-y
-      :close-on-content-click="false"
-      :nudge-width="350"
-      v-model="showCaseMenu"
+      <v-popover
+              v-model="showCaseMenu"
       >
-      <template v-slot:activator="{ on }">
-        <v-btn text v-on="on" v-if="caseSummary && caseSummary.name">
-          {{ caseSummary.name }}
-        </v-btn>
-      </template>
+        <button v-if="caseSummary && caseSummary.name"
+                offset-y
+        style="margin-bottom: 6px">          {{ caseSummary.name }}</button>
```

Comment:
Hi @WilliamRichards2017 
There is a blue border that is added when this button is clicked. 
<img width="157" alt="Screen Shot 2020-03-27 at 12 51 47 PM" src="https://user-images.githubusercontent.com/16284713/77796119-f5534080-702b-11ea-8591-9f74f53daa92.png">

Can we use the Vuetify button for consistency? 
The following code can fix it: 

```suggestion
          <v-btn v-if="caseSummary && caseSummary.name" text>{{ caseSummary.name }}</v-btn>
```